support/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Unreleased changes
 
+* [#468](https://github.com/mozilla-rally/rally-core-addon/pull/468): export rally run states instead of using strings.
```

Comment:
```suggestion
# v0.5.0 (2021-03-03)

* [#468](https://github.com/mozilla-rally/rally-core-addon/pull/468): export rally run states instead of using strings.
```